crates/bevy_time/Cargo.toml

```diff
@@ -20,7 +20,7 @@ bevy_ecs = { path = "../bevy_ecs", version = "0.14.0-dev", features = [
 ] }
 bevy_reflect = { path = "../bevy_reflect", version = "0.14.0-dev", features = [
   "bevy",
-] }
+], optional = true }
```

Comment:
Should we add a feature for enabling this dependency? And to control the feature flag for `bevy_ecs`? Right now it looks like `bevy_ecs` will always enable `bevy_reflect`.

Reply:
Do you mean something like adding a `bevy_reflect` feature to the root workspace that will enable another `bevy_reflect` feature in `bevy_internal`?

I was wondering why some crates have an optional dependency on `bevy_reflect` crate while enabling it in their defaults.

 I'm new to contributing to bevy.

Reply:
You need to add this to the cargo file. Assume the feature is called `bevy_reflect`. However, I think it should be renamed to `reflect` for short.
```yml
[features]
bevy_reflect = ["dep:bevy_reflect"]
```

Reply:
Thank you, I've just added it! Though, I noticed `bevy_app` also uses `bevy_reflect` by default.

Reply:
ah It need to be change to `default = ["bevy_reflect"]` instead. Sorry for the bad suggestion

Reply:
Oh, no problem! I'm changing the feature name right now.